src/AdminModule/Components/Navbar/Navbar.tsx

```diff
@@ -1,7 +1,40 @@
-import React from 'react'
+import { AppBar, Drawer, Toolbar, Typography } from "@mui/material";
+import React, { useState } from "react";
+import Tabs from "@mui/material/Tabs";
+import Tab from "@mui/material/Tab";
+import Button from "@mui/material/Button";
+
+import { Link, useNavigate } from "react-router-dom";
 
 export default function Navbar() {
+
+  const navigate = useNavigate();
   return (
-    <div>Navbar</div>
-  )
+    <>
+      <React.Fragment>
+        <AppBar sx={{ color: "#152C5B", background: "#fff" }}>
+          <Toolbar>
+            <Typography variant="h4" sx={{ marginLeft: "100px" }}>
+              <span className="text-primary ">Stay</span>cation.
+            </Typography>
+            <Tabs sx={{ marginLeft: "auto" }}>
+              <Tabs >
```

Comment:
We need these tabs to route to the actual pages, it is currently not working except for the Login.